### PR TITLE
Tutorial Introduction/Table of Contents Update

### PR DESCRIPTION
**Pull request by ACStoneCL**
### Related links

[https://casperlabs-team.slack.com/archives/C0135SDEGUC/p1649700128111589](https://casperlabs-team.slack.com/archives/C0135SDEGUC/p1649700128111589)

### Changes

As per request, I've updated the tutorial table of contents within the [Introduction](https://docs.casperlabs.io/tutorials/) page to include recently added tutorials.

I've also added the relevant tutorials to the [Writing Contracts Index](https://docs.casperlabs.io/writing-contracts/), which may have to be altered in the near-future as we revamp this content.

### Notes

Ran locally without issue.

### CODE REVIEW

**Review by ipopescu — DISMISSED** [review]
Thank you, @ACStoneCL! 

Is there a way to get this list automatically as it grows? It might not have detailed descriptions, but it would be nice to have this as the content grows.

**Comment by ACStoneCL** [comment]
@ipopescu I'm not sure about doing it automatically - I would assume it's possible, but it's definitely beyond my knowledge.

**Review by ipopescu — APPROVED** [review]
Thank you, @ACStoneCL. 
I don't know how to automatically display the links either. The list will need to be maintained manually (as it has been).
Also, there is duplication here, but the flow does get easier. So I will approve. Thanks for the quick turnaround!

**Comment by ipopescu** [comment]
@GloreenS, please take a look at this brief PR if you have time in the next couple of hours. If possible, we should merge it today.

File: source/docs/casper/dapp-dev-guide/tutorials/index.md [hunk 1]
```diff
@@ -9,8 +9,11 @@ The following tutorials present several Casper features in depth:
 
 | Title                                                       | Description                                                      |
 | ----------------------------------------------------------- | ---------------------------------------------------------------- |
-| [ERC-20 implementation](erc20/index.md)                     | Implement the ERC-20 standard for Casper                  |
-| [Key-value storage with Casper DSL](kv-storage-tutorial.md) | Design a simple contract to store a value and use the Casper DSL      |
-| [Multi-signatures and key recovery](multi-sig/index.md)     | Learn to sign transactions with multiple keys                    |
-| [Smart contract upgrades](upgrade-tutorial.md)              | Learn how to upgrade smart contracts  
-| [Casper Signer dApp integration](./dapp-dev-guide/tutorials/casper-signer)        |  Learn to use the Casper Signer to connect a wallet to your website  |
+|[A Counter Contract Tutorial](counter/index.md)              | A simple example contract that maintains a counter variable      |
```
Comment (GloreenS) [inline source/docs/casper/dapp-dev-guide/tutorials/index.md:12]: To be consistent with the titles of other tutorials can we change this to "Counter Contract Tutorial"?

**Review by GloreenS — DISMISSED** [review]
Couple minor suggestions, rest looks great!
- Unless we have a logical reason for the order in which the tutorials are listed, it would be nice to have them ordered alphabetically. 
- Punctuation in the right column should be consistent, either end with periods for all items or none at all. 
Thanks!

File: source/docs/casper/dapp-dev-guide/tutorials/index.md (hunk 1 above)
Comment (ipopescu) [reply source/docs/casper/dapp-dev-guide/tutorials/index.md:12]: I agree, thanks for noticing @GloreenS!

**Comment by ACStoneCL** [comment]
@GloreenS @ipopescu The order of the tutorials in the index came from the sidebar - but I agree it should be alphabetical. As such, I've alphabetized the sidebar as well as updating both of the new indexes. Also removed superfluous punctuation and renamed to "Counter Contract Tutorial" within the index.

File: config/sidebar.config.js [hunk 2]
```diff
@@ -91,7 +91,21 @@ module.exports = {
             collapsible: true,
             collapsed: true,
             items: [
-                "dapp-dev-guide/tutorials/index",
```
Comment (ipopescu) [inline config/sidebar.config.js:94]: It looks like we removed the index page by mistake. Fixing this in a new PR.

**Pull request merged**